priv/perf/apps/load_test/lib/common/byzantine_events.ex

```diff
@@ -191,9 +216,12 @@ defmodule OMG.Performance.ByzantineEvents do
   """
   @spec get_byzantine_events() :: list(map())
   def get_byzantine_events() do
-    watcher_url = Application.fetch_env!(:omg_performance, :watcher_url)
-    {:ok, status_response} = WatcherClient.get_status(watcher_url)
-    status_response[:byzantine_events]
+    {:ok, status_response} =
+      WatcherSecurityCriticalAPI.Api.Status.status_get(LoadTest.Connection.WatcherSecurity.client())
+
+    status = Jason.decode!(status_response.body)["data"]
+
+    status["byzantine_events"]
```

Comment:
seems brittle here. Should we be more defensive here? Also if you really want to go this route, may as well condense:

```suggestion
    status = Jason.decode!(status_response.body)["data"]["byzantine_events"]
```